### DISCUSSION TITLE
SEB Failed to start a new session! | The wait completed due to an abandoned mutex.

### DISCUSSION THREAD

**Discussion by Dhinesh-Dhinesh** [SafeExamBrowser/seb-win-refactoring#974] [Q&A] [answered]
![{E8C05D55-03E0-43F1-B78E-6B78B46C3B1C}](https://github.com/user-attachments/assets/34acfc2d-f37c-4c53-a224-d96d5e66d8f6)

![{B62DBB4A-9D71-44A2-93D4-517A935EEC73}](https://github.com/user-attachments/assets/ed6c61e9-52fc-465a-b362-598966f375b5)

Here is my log files
[runtime log.txt](https://github.com/user-attachments/files/17009839/runtime.log.txt)
[service log.txt](https://github.com/user-attachments/files/17009840/service.log.txt)

I'm encountering an issue with Safe Exam Browser (SEB) on my laptop, and despite trying several troubleshooting steps, the error still persists. Here's what I've done so far:

- Installed the latest Visual C++ Redistributable (VC Redist).
- Created a new profile on my laptop to check if the issue is profile-specific.
- Turned off all antivirus software to avoid potential conflicts.
- Ran SEB as an administrator.
- Reinstalled SEB several times.
- Confirmed that my Windows is fully updated with the latest patches.

Despite all these efforts, I continue to get the same error, and I can't seem to figure out what's wrong.

Has anyone else faced a similar issue or can provide any suggestions for fixing this? Any guidance would be greatly appreciated. Thanks!

**Comment by mastnym** [comment, marked as answer]
We are having same issue on several computers in our classroom. We are on 3.6 SEB version, WIN 11. What is very strange, this error pops up after lets say 10minutes of exam and only on several classroom computers.
Could this be the situation that a student is nervous before the exam and clicks the browser icon / associated config file multiple times?

Thanks Martin
